kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/AadSpec.java

```diff
@@ -25,7 +25,7 @@ public enum AadSpec {
     /**
      * No AAD
      */
-    NONE((byte) 0, EncryptionVersion.V1) {
+    NONE((byte) 0, EncryptionVersion.V2) {
```

Comment:
The intent behind coupling the AAD spec to the encryption version like this is so that once both are configurable by the user we have a way of knowing the encryption version required by a given AAD spec. So for example, AAD=batch metadata would require using EV >= 2.

So this shouldn't change, because the AAD=none was available from version 1; adding a version 2 doesn't change that.

Reply:
Sweet, I've reintroduce a V1 EncryptionVersion and bumped the Parcel and Wrapper versions the same way. AadSpec now declares NONE is valid from `V1_UNSUPPORTED`.

```
EncryptionVersion
V1_UNSUPPORTED((byte) 1, ParcelVersion.V1_UNSUPPORTED, WrapperVersion.V1_UNSUPPORTED),
V2((byte) 2, ParcelVersion.V2, WrapperVersion.V2);
```